shared/src/business/entities/EntityConstants.ts

```diff
@@ -548,13 +564,27 @@ export const OBJECTIONS_OPTIONS_MAP = {
 };
 export const OBJECTIONS_OPTIONS = [...Object.values(OBJECTIONS_OPTIONS_MAP)];
 
+export const INTERNAL_OBJECTION_DOCUMENT_TYPES = new Set([
+  ...DOCUMENT_INTERNAL_CATEGORIES_MAP['Motion'].map(entry => {
+    return entry.documentType;
+  }),
+  'Application to Take Deposition',
+]);
+
+export const EXTERNAL_OBJECTION_DOCUMENT_TYPES = new Set([
+  ...DOCUMENT_EXTERNAL_CATEGORIES_MAP['Motion'].map(entry => {
+    return entry.documentType;
+  }),
+  'Application to Take Deposition',
+]);
```

Comment:
This code was repeated in several places, so I extracted it. Because it was only ever used for lookups, I decided to use a Set.